test/test_crop_selection_audit.py

```
@@ -0,0 +1,257 @@
+    for test in tests:
+        try:
+            if test():
+                passed += 1
+            else:
+                failed += 1
+        except Exception as e:
+            print(f"❌ Ошибка в тесте {test.__name__}: {e}")
+            failed += 1
+    
+    print("\n" + "=" * 50)
+    print(f"📊 РЕЗУЛЬТАТЫ АУДИТА:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)